packages/react-data-grid/src/ReactDataGrid.js

```diff
@@ -66,6 +66,8 @@ const ReactDataGrid = React.createClass({
     onCellsDragged: React.PropTypes.func,
     onAddFilter: React.PropTypes.func,
     onGridSort: React.PropTypes.func,
+    sortColumn: React.PropTypes.string,
+    sortDirection: React.PropTypes.oneOf(['ASC', 'DESC', 'NONE']),
```

Comment:
Hi, i am facing one issue if i want to keep " link" inside the column may i know the process in react data grid.waiting your replay.